src/pages/Home/index.tsx

```diff
@@ -54,15 +56,19 @@ const Home: React.FunctionComponent<Props> = props => {
 					<ButtonCreateTeam onClick={toggleFormVisible}>
 						{t('createNewTeam')}
 					</ButtonCreateTeam>
-					{data.map(item => {
+					{data.map(team => {
 						const color = randomColor({
 							format: 'rgba',
 							alpha: 0.6,
 						})
 
+						const onTeamClick = () => {
```

Comment:
I think `handleClickTeam` or `onTeamClicked` would be better name

Reply:
Ok, I will change the name to handleClickTeam. I have just thought of default name onClick then it could be onTeamClick